### PR TITLE
BE-268: Added API for get teacher's step prices

### PR DESCRIPTION
**Pull request by shangerdi**
@Elors @Zack-Kang

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -1126,3 +1126,86 @@ HTTP Status Code 200
     "ok": true
 }
 ```
+
+### Get Teacher Prices
+
+```
+GET /api/v1/step_prices/
```
Comment (1e0ng) [inline docs/api.md]: You may use `/api/v1/teacher/1/school/1/prices` as the url so that no need to add parameters.

File: docs/api.md [hunk 2]
```diff
@@ -1126,3 +1126,86 @@ HTTP Status Code 200
...
+    "next": null,
+    "previous": null,
+    "results": [
+        {
+            "grade": 2,
+            "grade_name": "一年级",
+            "prices": [
+                {
+                    "min_hours": 1,
+                    "max_hours": 10,
```
Comment (1e0ng) [inline docs/api.md:1151]: Still need `max_hours`?

**Comment by zhoukang0107** [comment]
lgtm

File: docs/api.md (hunk 1 above)
Comment (shangerdi) [reply docs/api.md]: done

File: docs/api.md (hunk 2 above)
Comment (shangerdi) [reply docs/api.md:1151]: Will fix on next release

**Comment by liumengjun** [comment]
lgtm

**Pull request merged**